Musescore.config

```diff
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
```

Comment:
I copied that into the file .git/info/exclude - possibly that wasn't the correct place. No I saved it as "excludes".
How can I proceed to get the files away from the branch, does a new commit change that?